### PR TITLE
田同轩 1950081

### PR DESCRIPTION
**Pull request by Mocha259**
田同轩 1950081

### CODE REVIEW

File: task 2/main.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <opencv2\opencv.hpp>
```
Comment (ZhengQiushi) [inline task 2/main.cpp:2]: what's the difference between `\` and `/` ?

File: task 2/main.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#include<opencv2/imgproc/imgproc.hpp>
+#include<opencv2/highgui/highgui.hpp>
+
+using namespace cv;
+using namespace std;
+
+int main() 
+{
+	//namedWindow("test_image", CV_WINDOW_AUTOSIZE);
```
Comment (ZhengQiushi) [inline task 2/main.cpp:11]: please remove the meaningless comments in your code.

**Review by ZhengQiushi — COMMENTED** [review]
we are glad that you have noticed details about the coding style, keep on your good work.

File: task 2/main.cpp (hunk 1 above)
Comment (Mocha259) [reply task 2/main.cpp:2]: my carelessness
it's wrong in linux 
but allowed in windows
N.E.V.E.R  USE  " \ "

**Pull request closed without merging**